pages/common/astyle.md

```diff
@@ -0,0 +1,24 @@
+# astyle
+
+> Source code indenter, formatter, and beautifier for the C, C++, C# and Java programming languages.
+> A copy of the original file is created with an dot orig appended to the original file name.
+
+- Apply default style, 4 spaces per indent and no formatting changes:
+
+`astyle {source_file}`
+
+- Apply java style, attached braces:
+
+`astyle  --style=java {path/to/file}`
+
+- Apply allman style, broken braces:
+
+`astyle --style=allman {path/to/file}`
+
+- Apply custom indent using spaces, between 2 and 20:
+
+`astyle --indent=spaces={number_of_spaces} {path/to/file}`
```

Comment:
Did you mean to have an extra equals after the work `spaces`?

Reply:
Yes,
<img width="830" alt="screen shot 2017-10-13 at 10 53 43 pm" src="https://user-images.githubusercontent.com/6373522/31573093-7c0c2e64-b069-11e7-87a2-12c74890f0d0.png">
 that is how it appears in the documentation.

Reply:
Ahhh ok! Just checking :-)